test/GetEventStore.Tests/ProvidersFixture.cs

```diff
@@ -0,0 +1,47 @@
+﻿using EventSourcing.Tests;
+using Orleans.EventSourcing;
+using Orleans.Storage;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TestExtensions;
+
+namespace GetEventStore.Tests
+{
+    // creates several instance of a GetEventStore provider, with various parameter choices
+    public class ProvidersFixture : EventStorageProviderFixture
+    {
+
+        public readonly IEventStorageProvider EventStoreDefault;
+        public readonly IEventStorageProvider EventStoreSmallPageSize;
+
+        public readonly IEventStorageProvider EventStoreAllTypenames;
+
+        public readonly IEventStorageProvider EventStoreObjectIdentity;
+
+
+        public ProvidersFixture(TestEnvironmentFixture groupFixture) : base(groupFixture)
+        {
+
+            EventStoreDefault = new GetEventStoreProvider();
+            this.InitProvider(EventStoreDefault, "StandardPageSize");
```

Comment:
Since this depends on the environment being correctly set up, we'd need to detect if GetEventStore is not running, and skip the set up marking the fixture as invalid.
Then the constructor for the test methods that use this should check if it's invalid, and throw as SkipException in that case.
Then all Fact attributes should be changed to SkippableFact.

Search for SkippableFact usages if this wasn't clear, as we do this with every external/environmental dependency :)